pyhf/tensor/mxnet_backend.py

```diff
@@ -339,3 +350,31 @@ def log_prob(value, loc, scale):
                 scale, Number) else scale.log()
             return -((value - loc) ** 2) / (2 * variance) - log_scale - math.log(math.sqrt(2 * math.pi))
         return self.exp(log_prob(x, mu, sigma))
+
+    def normal_cdf(self, x, mu=[0], sigma=[1]):
+        """
+        The cumulative distribution function for the Normal distribution
+
+        Example::
+
+            >>> pyhf.tensorlib.normal_cdf([0.8])
+
+            [0.7881446]
+            <NDArray 1 @cpu(0)>
+
+
+        Args:
+            x (`tensor` or `float`): The observed value of the random variable
+                                      to evaluate the CDF for
+            mu (`tensor` or `float`): The mean of the Normal distribution
+            sigma (`tensor` or `float`): The standard deviation of the Normal distribution
+
+        Returns:
+            MXNet NDArray: The CDF
+        """
+        # This is currently using SciPy stats until a better way can be found to
+        # do this in MXNet
+        x = self.astensor(x).asnumpy().tolist()[0]
+        mu = self.astensor(mu).asnumpy().tolist()[0]
+        sigma = self.astensor(sigma).asnumpy().tolist()[0]
+        return self.astensor([norm.cdf(x, loc=mu, scale=sigma)])
```

Comment:
`norm` should be able to deal with numpy arrays, so probably don't need `tolist`

Reply:
Fixed in 0b76619